__tests__/app.test.js

```diff
@@ -163,13 +164,155 @@ test("200: responds with updated votes", () => {
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                comment_count: expect.any(Number),
+              })
+            )
+           })
+          })
+        })
+        
+  test("200: responds with articles in descending order", () => {
+    return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response)=> {
+          const {
+            _body: { article },
+          } = response;
+          expect(article).toBeSortedBy(`created_at`, {
+            descending: true
+          })
+        })
+      })
+      
     })
-})
-})
\ No newline at end of file
+
+    describe("9. GET: /api/articles/:article_id/comments", () => {
+      test("200: responds with correct (updated) article with comments_count", () => {
+       return request(app)
+        .get("/api/articles/5/comments")
+        .expect(200)
+        .then((response) => {
+          const {
+            body: { article },
+          } = response;
+          expect(article).toEqual([
+            {
+              comment_id: 15,
+              body: "I am 100% sure that we're not completely sure.",
+              author: 'butter_bridge',
+              created_at: '2020-11-24T00:08:00.000Z',
+              votes: 1
+            },
+            {
+              comment_id: 14,
+              body: 'What do you see? I have no idea where this will lead us. This place I speak of, is known as the Black Lodge.',
+              author: 'icellusedkars',
+              created_at: '2020-06-09T05:00:00.000Z',
+              votes: 16
+            }
+          ])
+        })
+      })
+
+      test("200: responds with the comments in descending order by date", () => {
+        return request(app)
+         .get("/api/articles/5/comments")
+         .expect(200)
+         .then((response) => {
+           const {
+             body: { article },
+           } = response;
+           expect(article).toBeSortedBy(`created_at`, {
+             descending: true
+           })
+         })
+       })
+       
+       test("404: responds with error for a route that is not available although is valid", () => {
+        return request(app)
+          .get("/api/articles/200/comments")
+          .expect(404)
+          .then(({body})=> {
+            expect(body.msg).toBe("This article does not exist")
+          })
+    })
+    })
+
+
+    describe("10. POST /api/articles/:article_id/comments",() => {
+    test("201: responds with newly created object containing correct keys", () => {
+      const newComment = {
+        username: 'icellusedkars',
+        body: 'This is impossible'
+      }
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then((response => {
+          const {
+            body: { article },
+          } = response
+          expect(article).toEqual(
+            expect.objectContaining({
```

Comment:
I this is a case where you could tests specifically what the object coming back looks like. I would use to equal here rather than objectContaining. This way I can check that it has the specific data I have sent in.